rectools/models/rank.py

```diff
@@ -156,7 +166,9 @@ def rank(
             If given, only these items will be used for recommendations.
             Otherwise all items from dataset will be used.
         num_threads : int, default 0
-            Will be used as `num_threads` parameter for `implicit.cpu.topk.topk`.
+            Will be used as `num_threads` parameter for `implicit.cpu.topk.topk`. Omitted if use_gpu is True
+        use_gpu : bool, default False
```

Comment:
Please update changelog: "Added `use_gpu` argument to `ImplicitRanker.rank` method"

Reply:
Done